docs/sources/operators-guide/deploy-grafana-mimir/getting-started-helm-charts/_index.md

```diff
@@ -79,6 +79,11 @@ Using a custom namespace solves problems later on because you do not have to ove
          # empty, disabled.
    ```
 
+   > **Note:** You can use `helm show values grafana/mimir-distributed` to see all configurable parameters for helm install.
+   > However, refrain from copy and paste the whole thing because all parameters default value are already set and you should only
```

Comment:
```suggestion
   > **Note:** To see all of the configurable parameters for a Helm chart installation, use `helm show values grafana/mimir-distributed`.
   > However, refrain from copying and pasting the whole configuration, because all of the parameters’ default values are already set. Only
```